src/sentry/index.ts

```diff
@@ -0,0 +1,20 @@
+import * as Sentry from "@sentry/browser";
+
+export const initializeSentry = () => {
```

Comment:
In the import statement, it is clear that this function is from sentry, as shown below, so the name of the function can be concise.

```ts
import { initialize as initializeSentry } from "./sentry";
```

```suggestion
export const initialize = () => {
```